src/utils/Api.js

```diff
@@ -0,0 +1,136 @@
+class Api {
+  constructor({ baseUrl, authToken }) {
+    this._baseUrl = baseUrl;
+    this._authToken = authToken;
+  }
+
+  getUsers = () =>
+    fetch(`${this._baseUrl}/users/me`, {
+      method: "GET",
+      headers: {
+        authorization: this._authToken,
+        "Content-Type": "application/json",
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      // if the server returns an error, reject the promise
+      return Promise.reject(`Error: ${res.status}`);
+    });
+
+  editUserInfo = (name, about) =>
+    fetch(`${this._baseUrl}/users/me`, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json", // Include Content-Type header
+        authorization: this._authToken,
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)